core/jazz_metrics/components/response.js

```diff
@@ -23,6 +23,7 @@
 **/
 
 module.exports = (response, input) => {
+  delete input["platform"];
```

Comment:
Why?

Reply:
Intended to keep the response for AWS metrics data unchanged. No hurt leaving the `platform` in the response. We can go either, unless we have a plan to display the `platform` on the UI in which we have to keep it.

Reply:
In java, we use transient to hide fields during json serialization.  Maybe this json ignore is working the same as transient.
https://www.npmjs.com/package/json-ignore